### ISSUE TITLE
P2P Network architecture 

### ISSUE THREAD

**Issue by Semenkoe** [Hive2Hive/Hive2Hive#153]
Hi,
At the moment I want to try to use it for my studying project and I have a question: (I didn't find the answer on TomP2P page, so probably you may know). What is the routing (network) distributed architecture is at the base of the framework? Is it Chord or Kademlia (or maybe something else)? 
Thank you in advance and thanks for the framework 👍

**Comment by nicoruti** [comment]
Hi @Semenkoe 
Thanks for your interest in TomP2P and Hive2Hive! TomP2P uses Kademlia routing concept.
Here are some slides of introduction lessons of TomP2P from the University: 

- https://tomp2p.net/doc/M05-1up_v2.pdf
- https://tomp2p.net/doc/M06-1up_v2.pdf

I hope they help you in understanding. Feel free to contribute (fork or pull request) on either project.

**Comment by nicoruti** [comment]
Yes, unfortunately, we had to stop developing on Hive2Hive due to time reasons.
What we have for documentation is:

- Wiki page: https://github.com/Hive2Hive/Hive2Hive/wiki
- Examples: https://github.com/Hive2Hive/Hive2Hive/tree/master/org.hive2hive.examples

Yes, everything we have is here on Github.
